studio/stores/UiStore.ts

```diff
@@ -157,4 +168,29 @@ export default class UiStore implements IUiStore {
   setPermissions(permissions?: any) {
     this.permissions = permissions
   }
+
+  /*
+   * Check wether the owner is allowed to leave a project
+   * the conditions for this is that they are an owner, and there is also 1 other owner in the org.
+   */
+  isOwnerAndCanLeaveOrg(members: Member[]) {
+    const selectedOrg = this.selectedOrganization
+
+    const roles: { roles: Role[] } = useOrganizationRoles(this.selectedOrganization?.slug)
+
+    const membersWhoAreOwners = members.filter((member) => {
+      const [memberRoleId] = member.role_ids ?? []
+      const role = (roles.roles || []).find((role: Role) => role.id === memberRoleId)
+      // dangerous use of string here
+      // to do, refactor so that changing the role name does not fail this check
+      // if role name does change, then user will simply not be able to leave project as a 'owner' regardless
+      return role && role.name === 'Owner' && !member.invited_at
+    })
+
+    console.log('membersWhoAreOwners', membersWhoAreOwners)
+
+    return selectedOrg?.is_owner && membersWhoAreOwners && membersWhoAreOwners.length > 1
```

Comment:
just `return selectedOrg?.is_owner && membersWhoAreOwners && membersWhoAreOwners.length > 1` is enough actually, don't need `? true : false`